src/liblsquic/lsquic_http1x_if.c

```diff
@@ -396,7 +399,15 @@ add_real_header (struct header_writer_ctx *hwc, struct lsxpack_header *xhdr)
     val_len = xhdr->val_len;
 
     if (4 == name_len && 0 == memcmp(name, "host", 4))
+    {
+    	if(hwc->pseh_mask & BIT(PSEH_AUTHORITY))
+		{
```

Comment:
Please use 4 spaces instead of TAB for indentation to make code look consistent. Same for all code changes.

Reply:
done